src/transaction/transactionAsyncApi.test.ts

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright 2021 RelationalAI, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { readFileSync } from 'fs';
+import nock from 'nock';
+
+import { baseUrl, getMockConfig } from '../testUtils';
+import { TransactionAsyncState } from '../transaction/types';
+import { TransactionAsyncApi } from './transactionAsyncApi';
+
+const path = '/transactions';
+
+const multipartMock = readFileSync(__dirname + '/multipartMock');
+const multopartContentType =
+  'multipart/form-data; boundary=865f68321ad9001da3f3b61785a72b25';
+const transactionAsyncMock = {
+  id: '2b23ad8b-f94d-4194-aab0-5ac2aa82d067',
+  state: 'COMPLETED',
+  problems: [],
+  metadata: [
+    { relationId: '/:output/:foo', types: [':output', ':foo'] },
+    { relationId: '/:output/Int64', types: [':output', 'Int64'] },
+  ],
+  relations: [
+    { relationId: '/:output/:foo', table: expect.anything() },
+    { relationId: '/:output/Int64', table: expect.anything() },
+  ],
```

Comment:
After thinking more about this: i think maybe we should call this section `results`, not `relations`, since it maps to the `GET /transactions/{id}/results` API endpoint.

I'm making that change in the julia SDK PR now. :)

Reply:
@NHDaly ok, I will